src/services/navigation/useNavLinks/useStaticNavLinks.test.tsx

```diff
@@ -60,7 +60,7 @@ describe('useStaticNavLinks', () => {
       ${links.uploaderCLI}                   | ${'https://docs.codecov.com/docs/codecov-uploader#using-the-cli-to-upload-reports-with-codecovio-cloud'}
       ${links.integrityCheck}                | ${'https://docs.codecov.com/docs/codecov-uploader#integrity-checking-the-uploader'}
       ${links.codecovGithubApp}              | ${'https://github.com/apps/codecov'}
```

Comment:
maybe replace this one too? Not sure where used

Reply:
Oh yeah good catch. I updated to pull from the var (which is set to `codecov` by default). It's used in some banners it looks like - I won't hide in those cases since the close-enough link is probably better than no link